gabbi/handlers/jsonhandler.py

```diff
@@ -86,6 +88,13 @@ def action(self, test, path, value=None):
         except ValueError:
             raise AssertionError('json path %s cannot match %s' %
                                  (path, test.response_data))
+
+        # read data from disk if the value starts with '<@'
+        if isinstance(value, str) and value.startswith('<@'):
+            info = test.load_data_file(value.replace('<@', '', 1))
```

Comment:
Also prefer @jd's approach, especially since the line above uses 'startswith', making the intent clear.